Core.Collectors/Config/StorageManager.cs

```diff
@@ -78,16 +78,16 @@ public StorageManager(JArray recordWritersArray, ITelemetryClient telemetryClien
         {
             JToken rootContainerToken = recordWriterToken.SelectToken("RootContainer");
             JToken notificationQueuePrefixToken = recordWriterToken.SelectToken("NotificationQueuePrefix");
-            JToken storageConnectionEnvironmentVariableToken = recordWriterToken.SelectToken("StorageConnectionEnvironmentVariable");
-            if (rootContainerToken == null || notificationQueuePrefixToken == null || storageConnectionEnvironmentVariableToken == null)
+            JToken storageAccountEnvironmentVariableToken = recordWriterToken.SelectToken("StorageAccountEnvironmentVariable");
+            if (rootContainerToken == null || notificationQueuePrefixToken == null || storageAccountEnvironmentVariableToken == null)
             {
```

Comment:
I did not expect this change as part of ROS. SplitAzureBlobWriter is actually used by modern collectors, so there is a mismatch, at least conceptually. Could you please ping me so that we can clarify the confusion?

Reply:
Will make these changes as part of future release